dashboard/app/controllers/programming_expressions_controller.rb

```diff
@@ -42,8 +43,9 @@ def update
       render :not_found
       return
     end
-    programming_expression.name = programming_expression_params[:name]
-    programming_expression.short_description = programming_expression_params[:short_description]
+    programming_expression_params.each do |key, val|
+      programming_expression.send("#{key}=", val)
+    end
```

Comment:
cool pattern! would you consider a simpler option though, assuming this actually works?
```suggestion
    programming_expression.assign_attributes(programming_expression_params)
```
https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update-21 --> "Source: show" suggests that `update!` consists of `assign_attributes` followed by `save!`, so it seems like this is roughly what we want.

Reply:
I thought about assign_attributes but I thought it wouldn't work because of the `properties` field on our models. But let me double check because I think that would be much simpler!

Reply:
I am wrong! I'll use `assign_attributes` here :)